packages/react-client/src/ReactFlightPerformanceTrack.js

```diff
@@ -412,7 +412,13 @@ function getIOShortName(
       const slashIdx = description.lastIndexOf('/', queryIdx - 1);
       if (queryIdx - slashIdx < descMaxLength) {
         // This may now be either the file name or the host.
-        desc = ' (' + description.slice(slashIdx + 1, queryIdx) + ')';
+        // Include the slash to make it more obvious what we trimmed.
+        desc = ' (…' + description.slice(slashIdx, queryIdx) + ')';
+      } else {
+        // cut out the middle to not exceed the max length
```

Comment:
Not sure this is better. The max length is assuming that most things will be either below the limit or excluded all together. This makes it so that much more is always the exact limit which is large. Making it much noisier.

Additionally, Chrome will insert its own ellipsis in the middle when it's collapsed which can lead to two.

Reply:
I think it would be better to just elide it if it's long or increase the limit and let Chrome insert the ellipsis in the middle.

Reply:
> Additionally, Chrome will insert its own ellipsis in the middle when it's collapsed which can lead to two.

I guess the original idea was that we don't want to make the track too busy so a limit does make sense. The double ellipsis is unfortunate. Eliding the value felt pretty bad. It looked like some information was missing.